src/components/forms/VolumeSnapshotsForm.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, ReactNode } from "react";
+import { Input } from "@canonical/react-components";
+import InstanceConfigurationTable from "components/forms/InstanceConfigurationTable";
+import SnapshotScheduleInput from "components/SnapshotScheduleInput";
+import { getStorageConfigurationRow } from "pages/storage/forms/StorageConfigurationRow";
+import { useDocs } from "context/useDocs";
+import { FormikProps } from "formik";
+import { StorageVolumeFormValues } from "pages/storage/forms/StorageVolumeForm";
+
+interface Props {
+  formik: FormikProps<StorageVolumeFormValues>;
+  children?: ReactNode;
+}
+
+const VolumeSnapshotsForm: FC<Props> = ({ formik }) => {
+  const docBaseLink = useDocs();
+  return (
+    <InstanceConfigurationTable
+      rows={[
+        getStorageConfigurationRow({
+          formik,
+          label: "Snapshot name pattern",
+          name: "snapshots_pattern",
```

Comment:
A custom pattern is not respected when creating new snapshots. This might be a backend bug. Can you investigate and communicate with the core team on this?

To reproduce: change the snapshot pattern to "snappy%d" on a volume. Create a snapshot, leave all fields default (especially empty name). The snapshot is created as "snap1".
Doing the same thing on an instance, results in a snapshot with the name "snappy1".

Reply:
An issue will be created for the core team to resolve this.